fabric/bin/lib/common_ledger.sh

```diff
@@ -112,15 +112,32 @@ ledger_init() {
     try rm -rf ${FABRIC_STATE_DIR}/*
 
     # 2. start orderer
+    # try to create orderer-system-channel genesisblock
+    para
+    echo "[+] Creating a genesisblock for orderer-system-channel......."
+    para
+    try ${CONFIGTXGEN_CMD} -outputBlock genesisblock -profile SampleDevModeSolo -channelID orderer-system-channel
```

Comment:
Given that solo and kafka is now depricated, we probably should move to Raft (and hence probably the SampleDevModeEtcdRaft profile). From what i understood from Swetha Raft seems also faster that solo, so might be win-win. @MHBauer probably can advise whether just changing the genesis profile is enough and/or whether there are twists with raft which maybe should keep us stick with solo for the time being ...

Reply:
General concern - How can solo by faster than RAFT? ... taking about protocol not implementation - WIRED

Reply:
> General concern - How can solo by faster than RAFT? ... taking about protocol not implementation - WIRED

not following ...

Reply:
btw: seems just simply replacing `SampleDevModeSolo` by `SampleDevModeEtcdRaft` does not work as raft seems to require some additional setup with certs or alike:
```
Error on outputBlock: could not create bootstrapper: could not create channel group: could not create orderer group: cannot marshal metadata for orderer type etcdraft: cannot load client cert for consenter raft0.example.com:7050
```
....

Reply:
figured out the problem: you didn't properly comment out the decorator by leaving the leading '-'.  Pretty ugly, though, that this causes the peer to crash in a SIGSEG for a invalidly formatted config file !!